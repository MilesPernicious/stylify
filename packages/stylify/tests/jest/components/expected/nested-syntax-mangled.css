.n:hover{
	text-decoration: none
}
.p{
	text-decoration: none
}
.q{
	display: flex
}
#wrapper .not-minified-2,
.l .not-minified,
.r{
	color: darkred
}
#wrapper .m,
.s{
	color: green
}
#wrapper header,
.b+input[type='text'],
.c .orange,
header .l,
.t{
	color: orange
}
header .a,
.l span,
.u{
	font-weight: bold
}
.b+input[type='text'],
.v{
	margin-left: 24px
}
.d .orange.large,
.w{
	font-size: 48px
}
.e>div,
.f span,
.x{
	color: blue
}
header .g>span,
.m + #header,
.y{
	color: purple
}
.h:hover span,
.z{
	text-decoration: underline
}
.i div,
.aa{
	width: 24px
}
.i div,
.ab{
	height: 24px
}
.i div,
.ac{
	background: blue
}
.j div:last-of-type,
.j div:nth-child(2):hover,
.j div:first-of-type,
.ad{
	background: black
}
.k,
.ae{
	font-size: 24px
}
.k span,
.af{
	border-left: 4px solid red
}
.l,
.ag{
	background: #000
}
.l,
.ah{
	color: #fff
}
.l,
.ai{
	border: none
}
.l,
.aj{
	transition: background 0.3s
}
.l:hover,
.ak{
	background: #888
}
.l > span,
.l >  i,
.al{
	vertical-align: middle
}
header .l,
.am{
	padding: 12px
}
.l.k,
.an{
	padding: 24px
}
.l.k span,
.ao{
	border-left: 8px solid red
}
.m,
.ap{
	font-style: italic
}

@media (min-width: 768px) {
header .l,
.o{
	margin: 12px
}
}
