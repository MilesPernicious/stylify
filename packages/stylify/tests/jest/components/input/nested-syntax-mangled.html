<header><div class="[header_&]{font-weight:bold}">Bold</div></header>

<div class="[&+input[type=^text^]]{margin-left:24px;color:orange} [.orange]{color:orange} [.orange.large]{font-size:48px}">
	<div class="orange">Orange</div>
	<div class="orange large">Orange large</div>
</div>

<input type="text">

<div class="[&>div]{color:blue} [&_span]{color:blue}">
	<div>Blue div</div>
	<span>Blue span</span>
	<main>
		<div>Normal div</div>
	</main>
</div>

<header>
	<div class="[header_&>span]{color:purple}">
		<span>Purple</span>
		<div><span>Normal</span></div>
	</div>
</header>

<a href="#" class="[&:hover_span]{text-decoration:underline} text-decoration:none hover:text-decoration:none">
	<b>Icon</b>
	<span>Underlined</span>
</a>

<div>
	<button class="button">Button</button><br>
	<button class="button button--big">Button big<span>Border red overriden</span></button><br>
	<button class="button">
		<i>icon</i>
		<span>Span</span>
	</button><br>
	<header>
		<button class="button">Button orange</button>
	</header>
</div>

<div class="display:flex [div]{width:24px;height:24px;background:blue} [div:last-of-type,div:nth-child(2):hover,div:first-of-type]{background:black}">
	<div></div>
	<div></div>
	<div></div>
</div>

<div id="#wrapper">
	<div class="header">Green</div>
	<div id="header">Purple</div>
	<header>Orange</header>
</div>
