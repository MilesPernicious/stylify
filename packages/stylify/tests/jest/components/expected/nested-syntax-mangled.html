<header><div class="a">Bold</div></header>

<div class="b c d">
	<div class="orange">Orange</div>
	<div class="orange large">Orange large</div>
</div>

<input type="text">

<div class="e f">
	<div>Blue div</div>
	<span>Blue span</span>
	<main>
		<div>Normal div</div>
	</main>
</div>

<header>
	<div class="g">
		<span>Purple</span>
		<div><span>Normal</span></div>
	</div>
</header>

<a href="#" class="h p n">
	<b>Icon</b>
	<span>Underlined</span>
</a>

<div>
	<button class="l">Button</button><br>
	<button class="l k">Button big<span>Border red overriden</span></button><br>
	<button class="l">
		<i>icon</i>
		<span>Span</span>
	</button><br>
	<header>
		<button class="l">Button orange</button>
	</header>
</div>

<div class="q i j">
	<div></div>
	<div></div>
	<div></div>
</div>

<div id="#wrapper">
	<div class="m">Green</div>
	<div id="header">Purple</div>
	<header>Orange</header>
</div>
